cms/templates/base.html

```diff
@@ -85,7 +85,7 @@
                       url_name: "${context_course.location.name | h}",
                       org: "${context_course.location.org | h}",
                       num: "${context_course.location.course | h}",
-                      display_course_number: "${_(context_course.display_number_with_default)}",
+                      display_course_number: "${_(context_course.display_coursenumber)}",
```

Comment:
As per my findings, @saleem-latif made the same change in PR#10075 , @mattdrayer you can see the base.html file in [PR#10075](https://github.com/edx/edx-platform/pull/10075). 
@saleem-latif added the corresponding condition to support his changes at  [LINE](https://github.com/edx/edx-platform/blob/master/cms/templates/js/certificate-details.underscore#L40). 
However, I had discussed this change with TNL Team here, I think @muhammad-ammar can point out if anything is going wrong the above change.